pageserver/src/config.rs

```diff
@@ -87,6 +87,8 @@ pub mod defaults {
 
     pub const DEFAULT_GET_VECTORED_IMPL: &str = "sequential";
 
+    pub const DEFAULT_MAX_VECTORED_READ_SIZE: usize = 128 * 1024; // 128 KiB
```

Comment:
Maybe `_bytes` instead of `_size`?  (in most codebases size is implicitly bytes, but we occasionally have sizes in pages)

Reply:
https://github.com/neondatabase/neon/pull/6780/commits/4b0a9a58a9b195c26d60f59b9feb180abcefa848